README.md

```diff
@@ -1,67 +1,142 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: azure/login@v1                            # Azure login required to add a temporary firewall rule
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+    - uses: azure/sql-action@v1.2
+      with:        
+        connection-string: ${{ secrets.AZURE_SQL_CONNECTION_STRING }}
+        sql-file: './sqlscripts/*.sql'
+```
+
+### Deploy a DACPAC to an Azure SQL database with Allow Azure Services access enabled
+
+```yaml
+# .github/workflows/sql-deploy.yml
+on: [push]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: azure/sql-action@v1.2
+      with:
+        connection-string: ${{ secrets.AZURE_SQL_CONNECTION_STRING }}
+        dacpac-package: './Database.dacpac'
+        arguments: '/p:DropObjectsNotInSource=true'   # Optional parameters for SqlPackage Publish
+```
+
+
+## 📓 User Guide
+
+### Authentication
+
+The v1.x version of sql-action supports SQL authentication only in the connection string.
+
+The `server-name` action YAML key is optional and is only available to provide backward compatibility. It is strongly recommended to put the server name in the connection string as displayed in the examples. The connection string uses this template: `Server=<servername>; User ID=<user_id>; Password=<password>; Initial Catalog=<database>`. In case the server name is put both in the `server-name` and in the `connection-string`, the server name used will be the one specified in the `server-name` YAML key.
+
+### Environments
+
+sql-action is supported on both Windows and Linux environments.  The [default images](https://github.com/actions/virtual-environments) include the prerequisites:
+
+- sqlcmd
+- sqlpackage (for sqlproj or dacpac deployment)
+- dotnet (for sqlproj build)
+
+### Firewall Rules/Access
+
+If you *can* use the option [Allow Azure Services and resources to access this server](https://docs.microsoft.com/en-us/azure/azure-sql/database/firewall-configure#connections-from-inside-azure), you are all set and you don't need to to anything else to allow GitHub Action to connect to your Azure SQL database.
 
-### Configure GitHub Secrets
+If you *cannot* use the aforementioned option, the action can automatically add and remove a [SQL server firewall rule](https://docs.microsoft.com/azure/sql-database/sql-database-server-level-firewall-rule) specific to the GitHub Action runner's IP address. Without the firewall rule, the runner cannot communicate with Azure SQL Database. **To enable the action to automatically add/remove a firewall rule, add an [Azure/login](ttps://github.com/Azure/login) step before the sql-action step.** Also, the service principal used in the Azure login action needs to have elevated permissions, i.e. membership in SQL Security Manager RBAC role, or a similarly high permission in the database to create the firewall rule.

```

Comment:
I don't really know Azure actions so maybe this feedback is coming from too unfamiliar of a mind, but I like to do something to indicate that some text is meant to be read/used literally rather than semantically.  E.g.

`Azure/login` (with code backticks) instead of Azure/login (no adornment).  Doublequotes work nicely, too.